test/functional/services/find.ts

```diff
@@ -28,25 +28,31 @@ export async function FindProvider({ getService }: FtrProviderContext) {
   const retry = getService('retry');
 
   const driver = webdriver.driver;
-  const By = webdriver.By;
   const until = webdriver.until;
   const browserType = webdriver.browserType;
 
   const WAIT_FOR_EXISTS_TIME = config.get('timeouts.waitForExists');
   const defaultFindTimeout = config.get('timeouts.find');
   const fixedHeaderHeight = config.get('layout.fixedHeaderHeight');
 
-  const wrap = (webElement: WebElement | WebElementWrapper) =>
+  const wrap = (webElement: WebElement | WebElementWrapper, locator: By | null = null) =>
     new WebElementWrapper(
       webElement,
+      locator,
       webdriver,
       defaultFindTimeout,
       fixedHeaderHeight,
       log,
       browserType
     );
 
-  const wrapAll = (webElements: Array<WebElement | WebElementWrapper>) => webElements.map(wrap);
+  const wrapAll = (webElements: Array<WebElement | WebElementWrapper>) =>
+    webElements.map(e => wrap(e));
```

Comment:
Could be `webElements.map(wrap` _pointfree style_ :)

Reply:
Wrap takes multiple arguments, and `.map()` passes its callback multiple arguments, this is why I basically always use an arrow function when calling `.map()`.